docs/lab/workflow/order-requirements.mdx

```diff
@@ -4,16 +4,115 @@ title: "Order Requirements"
+
+#### Spiriplex
+
+For placing insurance orders with Spiriplex, you will need some pre-requisites:
+
+- A physician capable of signing a requisition form. We currently don't support insurance orders using our own private physician network.
+- A clear, legible image of the front and back of the Patient insurance card
+- A clear, legible image of the Patient signature
+- Information about the patient reported symptoms, and physician assessment and treatment plan
+
+You can provide this information in our API, using the `physician` and `health_insurance` arguments for the [POST v3/order/](/api-reference/lab-testing/create-order) REST endpoint.
+
+```json Health Insurance
+{
+"health_insurance":{
+      "front_image": {
+         "content": "base64-encoded insurance front image binary content",
+         "content_type": "image/png | image/jpg | image/jpeg"
+      },
+      "back_image": {
+         "content": "base64-encoded insurance back image binary content",
+         "content_type": "image/png"
+      },
+      "patient_signature_image": {
+         "content": "base64-encoded patient signature image binary content",
+         "content_type": "image/png"
+      },
+      "subjective": "Description of what is the patient symptoms and attempted treatments",
+      "assessment_plan": "Description of what is the physician assessment and treament or testing plan"
+    }
+}
+```
+
+```json Physician
+{
+"physician": {
+    "first_name": "John",
+    "last_name": "Doe",
+    "email": "john@doe.com",
+    "npi": "123456789",
+    "licensed_states": ["CA", "NY"],
+    "signature_image": {
+         "content": "base64-encoded physician signature image binary content",
+         "content_type": "image/png"
+      }
+  }
+}
+```
+
+After you place an order with all the required information, a requisition form will be generated and sent to Spiriplex
+with the necessary information so your customer’s Health Insurance can be billed by Spiriplex.
+#### Labcorp
+
+To place an Order with Health Insurance as the billing method in Labcorp, you can use the `health_insurance` argument 
+for the [POST v3/order/](/api-reference/lab-testing/create-order) REST endpoint, providing the following required information:
+
+- A clear, legible image of the front and back of the Patient insurance card
+- Payor Code - A unique code representing a specific Health Insurance company. You can query a specific Payor Code using the [Search Payor route](/api-reference/lab-testing-insurance/search-payor)
+- Insurance ID - A unique alphanumerical number representing the patient contract with the Health Insurance company. Usually found in the patient's insurance card.
+- Responsible Relationship - The relationship between the Insurance owner and the patient. The values are (Self, Spouse, Other Relationship). If the relationship is different from "Self", the Responsible Details field must be provided.
```

Comment:
Self, Spouse and Other Relationship should be marked as code (e.g. Self -> `Self`), since these values are passed to an API.